src/server.js

```diff
@@ -1,22 +1,28 @@
 import Ganache from "ganache-core";
 import Web3 from "web3";
 import { Profiler, Contracts, Migrate } from "./truffle/external";
+import portfinder from "portfinder";
 
 export default class Server {
-  constructor() {
+  constructor(options = {}) {
     this.ganache = Ganache.server({
       default_balance_ether: 1000
     });
     this.web3 = new Web3();
     this.accounts = [];
+    this.startingPort = options.port || 8545;
```

Comment:
Is `this.startingPort` only used for the next line?

Reply:
I am removing it. Only had it because Paul had mentioned in one of the comments he wanted it.